build/preview.ts

```diff
@@ -0,0 +1,40 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+async function run() {
+  const cwd = path.resolve(__dirname);
+
+  const buildDir = path.basename(cwd);
+  const repoDir = cwd.replace(buildDir, '');
+
+  // Patch README
+  const data = fs.readFileSync(path.join(repoDir, 'README.md'), { encoding: 'utf8' });
+  const insert = fs.readFileSync(path.join(buildDir, 'nightly.md'), { encoding: 'utf8' });
+  fs.writeFileSync(path.join(repoDir, 'README.md'), `${insert}\n${data}`);
```

Comment:
This won't replace the `# Terraform Visual Studio Code Extension` headline in the `README.md` anymore, but I think that's fine and looks better now.

There is an old typo in `nightly.md` : `Preivew`. Do you mind fixing it?

Reply:
Nice catch, didn't notice the line not being replaced in my frustration at `git log` date formatting not working across platforms.

Reply:
With the new matrix approach, we no longer need this file